detection_rules/devtools.py

```diff
@@ -0,0 +1,203 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+
+"""CLI commands for internal detection_rules dev team."""
+import glob
+import io
+import json
+import os
+import shutil
+import subprocess
+
+import click
+from eql import load_dump
+
+from . import rule_loader
+from .main import root
+from .misc import PYTHON_LICENSE, client_error
+from .packaging import PACKAGE_FILE, Package, manage_versions, RELEASE_DIR
+from .rule import Rule
+from .utils import get_path
+
+
+RULES_DIR = get_path('rules')
+
+
+@root.group('dev')
+def dev_group():
+    """Commands for development and management by internal Elastic team."""
```

Comment:
idk if the internal verbiage is the best idea for this tool/repo, but I like where you're going with this